buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -559,6 +559,9 @@ class BeamModulePlugin implements Plugin<Project> {
 
       project.tasks.withType(JavaCompile) {
         options.encoding = "UTF-8"
+        // As we want to add '-Xlint:-deprecation' we intentionally remove '-Xlint:deprecation' from compilerArgs here,
+        // as intellij is adding this, see https://issues.apache.org/jira/browse/BEAM-5176
```

Comment:
I think the most useful link here might be https://youtrack.jetbrains.com/issue/IDEA-196615, but let's just interlink them and all that.

Reply:
Thinking about that, u are probably right. Should we change and force-push here?

Reply:
Yes, that would be fine. As it is just a comment change I won't have to wait another hour for tests.

Reply:
Done